experimenter/tests/integration/nimbus/test_desktop_targeting.py

```diff
@@ -15,9 +16,24 @@ def targeting_config_slug(request):
 
 
 @pytest.fixture
+@cache
 def filter_expression_path():
     path = Path(__file__).parent / "utils" / "filter_expression.js"
-    return path.absolute()
+    with path.open() as f:
+        return f.read()
+
+
+@pytest.fixture(scope="module")
+def browser_initialized():
+    return {"initialized": False}
+
+
+@pytest.fixture(autouse=True, scope="function")
+def setup_browser(selenium, browser_initialized):
+    if not browser_initialized["initialized"]:
+        selenium.get("about:blank")
+        browser_initialized["initialized"] = True
+    yield
```

Comment:
Why are we doing this extra layer of indirection?

```suggestion
@pytest.fixture(autouse=True, scope="module")
def setup_browser(selenium):
    selenium.get("about:blank")
    yield
```

Reply:
so changing it to module CI fails https://app.circleci.com/pipelines/github/mozilla/experimenter/65876/workflows/6adb259c-6aec-4d7c-8f2e-d68915ff4edd/jobs/404467 
https://github.com/mozilla/experimenter/pull/14084#discussion_r2572301731

Reply:
I have reverted to use the workaround

Reply:
since selenium is function-scoped, so to work around with this I did this

    Created browser_initialized module-scoped fixture to track state
    Modified setup_browser to only call selenium.get("about:blank") once per module
    Uses lazy initialization pattern to work around selenium's function scope